packages/chevrotain/src/scan/lexer.ts

```diff
@@ -85,14 +63,14 @@ export function analyzeTokenTypes(
     tracer?: (msg: string, action: () => void) => void
   }
 ): IAnalyzeResult {
-  options = defaults(options, {
+  options = {
     useSticky: SUPPORT_STICKY,
-    debug: false as boolean,
```

Comment:
This `debug` flag appeared unused and caused TypeScript to complain, so I removed it.